server/src/main/java/org/opensearch/tracing/Tracer.java

```diff
@@ -86,20 +65,30 @@ default Span startTrace(String spanName, Map<String, String> attributes, Level l
      * Level is an OpenSearch specific concept which
      */
     enum Level {
-        UNKNOWN(0), LOWEST(1), LOW(2), MID(3), HIGH(4);
+        TRACE(0), DEBUG(1), INFO(2), TERSE(3), ROOT(4), DISABLED(5);
         private final int order;
 
         Level(int order) {
             this.order = order;
         }
 
-        public boolean isHigherThan(Level level) {
```

Comment:
Let's keep this logic here. So that we don't replicate the same at multiple places.